standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/datasource/HikariCPDataSourceProvider.java

```diff
@@ -67,6 +67,7 @@ public DataSource create(Configuration hdpConfig) throws SQLException {
     config.setJdbcUrl(driverUrl);
     config.setUsername(user);
     config.setPassword(passwd);
+    config.setLeakDetectionThreshold(3600*1000);
```

Comment:
Can you get this value via Configuration object? (as in, make it configurable value)

Reply:
Fixed it